packages/mdc-select/mdc-select.scss

```diff
@@ -153,6 +155,16 @@
         @include mdc-theme-prop(color, text-primary-on-dark);
       }
     }
+
+    &--disabled {
```

Comment:
What do you think if we extract this part and make it a mixin class in either `mdc-simple-menu` or `mdc-list`? Then we may remove the duplicated definition here.